src/betterTableLayout.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @desc Combines mark and denominator into one cell
+ */
+const betterTableLayout = () => {
+    // fix table header
+    const row = $('#markbookTable tr:first');
+    const itemsHeader = row.find('td:nth-child(1)');
+    const marksHeader = row.find('td:nth-child(2)');
+    const dateHeader = row.find('td:nth-child(3)');
+    const weightHeader = row.find('td:nth-child(4)');
+
+    // reformat
+    row.empty();
+    row.append(itemsHeader, marksHeader, weightHeader, dateHeader);
+
+    // fix table content
+    $('#markbookTable tr:not(:first)').each(function () {
+        const row = $(this);
+        const name = row.find('td:nth-child(1)').clone(true, true);
+        const mark = row.find('td:nth-child(2)').clone(true, true);
+        const date = row.find('td:nth-child(3)').clone(true, true);
+        const weight = row.find('td:nth-child(4)').clone(true, true);
+        const denominator = $(this).find('td:nth-child(5)').clone(true, true);
+        
+        // set name column width
+        name.css('width', '350px');
```

Comment:
```suggestion
        name.css('width', '300px');
```
The original width is 300px, I understand that a column was removed but 350 is unnatural.
![image](https://user-images.githubusercontent.com/43765559/103140453-e7f2e180-46a3-11eb-80e6-e7d131d08cf8.png)

![image](https://user-images.githubusercontent.com/43765559/103140441-ceea3080-46a3-11eb-9446-d532a592323c.png)

Reply:
That's fair. The initial intention was for the other columns (weight, date) to have the same width as before. However, I guess it does look a little unnatural.